krogoth.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+	<remote name="github" fetch="https://github.com" pushurl="ssh://git@github.com" />
+	<remote name="yocto" fetch="https://git.yoctoproject.org/git/" />
+	<remote name="openembedded" fetch="git://git.openembedded.org/" />
+	<remote name="agl" fetch="https://gerrit.automotivelinux.org/gerrit/" />
+
+	<!-- Core layers: poky and meta-oe -->
+	<project path="poky" name="poky" remote="yocto" revision="krogoth" />
+	<project name="meta-openembedded" path="meta-openembedded" remote="openembedded" revision="krogoth" />
+
+	<!-- Updater layers -->
+	<project path="meta-updater" name="advancedtelematic/meta-updater" remote="github" revision="krogoth" />
+	<project path="meta-updater-raspberrypi" name="advancedtelematic/meta-updater-raspberrypi" remote="github" revision="krogoth" />
+	<project path="meta-updater-minnowboard" name="advancedtelematic/meta-updater-minnowboard" remote="github" revision="morty" />
```

Comment:
I would change revisions of all the meta-updater-* to krogoth for consistency. Just created dummy krogoth branches where they were missing.